pkg/p2p/streamtest/streamtest.go

```diff
@@ -153,6 +161,13 @@ func (r *Recorder) NewStream(ctx context.Context, addr swarm.Address, h p2p.Head
 	return streamOut, nil
 }
 
+func (r *Recorder) Ping(ctx context.Context, addr ma.Multiaddr) (rtt time.Duration, err error) {
```

Comment:
Given that the Recorder has a very good test coverage, a simple unit test for the new Ping method would be nice to have, even if it just calls the injected function. That would make the test even easier.